react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { button: true, switch: true, label: true };
```

Comment:
If all the values in state are either all `true` or all `false` couldn't you just have one property in state? Something like `isOff`.

Reply:
fixed